funcx_sdk/funcx/sdk/executor.py

```diff
@@ -263,89 +453,6 @@ def noop():
     return
 
 
-class ExecutorPollerThread:
```

Comment:
Was it strictly necessary to move this class at the same time as making changes to it? Can that change be reverted so that the diff is only new and removed lines? Right now, this is one of the reasons that it's hard to get a handle on these changes.